pyproject.toml

```
@@ -23,7 +23,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-async-timeout = "^4.0.1"
+async-timeout = ">=3.0.0"
```

Should we also include a max version to avoid potential breaking changes in the future?
```suggestion
async-timeout = ">= 3.0.0, < 5"
```